.github/workflows/release.yml

```
@@ -37,3 +37,16 @@ jobs:
         tag_name: ${{ steps.fetch-release.outputs.MICROMAMBA_VERSION }}
         files: |
           releases/*
+
+    - name: Prerelease
+      uses: softprops/action-gh-release@v1
+      if: steps.fetch-release.outputs.MICROMAMBA_NEW_PRERELEASE == 'true'
+      with:
+        name: "micromamba ${{ steps.fetch-release.outputs.MICROMAMBA_VERSION }}"
+        prerelease: true
```

```suggestion
      with:
        name: "micromamba ${{ steps.fetch-release.outputs.MICROMAMBA_VERSION }}"
        prerelease: ${{ steps.fetch-release.outputs.MICROMAMBA_NEW_PRERELEASE == 'true' }}
```

something like this should work as well